src/views/SignupPage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Link} from 'react-router-dom';
+import SignupForm from '../components/SignupForm';
+
+/**
+ * @class SignupPage
+ */
+class SignupPage extends React.Component {
+  /**
```

Comment:
#### Nitpick
- Lazy JS-DOCs, don't you think?

Reply:
Alright, thanks. https://github.com/andela/apollo-ah-frontend/blob/feature%2F164198142%2Fsignup/src%2Fviews%2FSignupPage.jsx/#L1-L14